### PR TITLE
[Issue 63] Created Linking endpoint between user and project

### PR DESCRIPTION
**Pull request by thhu**
Created endpoint to link user and project. Users contain a list of linked projects and Projects contain a list of linked users. 

issue link: https://github.com/SeattleVoluntech/seattlevoluntech/issues/63
pr: https://github.com/SeattleVoluntech/seattlevoluntech/pull/96

### CODE REVIEW

**Comment by FangJennyYuan** [comment]
Previous CR was closed too many merge conflicts, here is the new one

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -9,4 +9,7 @@ spring.datasource.username=${DB_USER}
 spring.datasource.password=${DB_PASS}
 
 #Do nothing
-spring.jpa.hibernate.ddl-auto=none
\ No newline at end of file
+#spring.jpa.hibernate.ddl-auto=none
+
+#Create tables in datasource.url if they do not exist
+spring.jpa.hibernate.ddl-auto=update
```
Comment (Richard-outreach) [inline src/main/resources/application.properties]: remove still :D

Comment (FangJennyYuan) [reply src/main/resources/application.properties]: whoops...

Comment (FangJennyYuan) [reply src/main/resources/application.properties]: fixed

Comment (FangJennyYuan) [reply src/main/resources/application.properties]: Also updated the schema.sql file

**Pull request merged**